src/items/hit-location-item/hitLocationSheet.ts

```diff
@@ -297,6 +297,13 @@ export class HitLocationSheet extends RqgItemSheet<
 
   static async setTokenEffect(token: RqgToken, actorHealthBefore: ActorHealthState): Promise<void> {
```

Comment:
This parameter isn't used anymore so it could be removed (along with it's calculation where it's called)
```suggestion
  static async setTokenEffect(token: RqgToken): Promise<void> {
```
Maybe you could also remove my stray leftover comment row "TODO testing testing..." as well :-)

Reply:
Done